jina/drivers/__init__.py

```diff
@@ -254,6 +255,7 @@ def pre_traverse(_docs, traverse_on, context_doc=None):
         else:
             raise ValueError(f'{self.recursion_order}')
 
+        # context_doc is missing, and this leads to `_apply_all` is skipped
```

Comment:
what does this mean? can you remove this comment if nothing wrong with the logic?

also, always refer to #702 illustration about the doc structure

Reply:
Some debug messages left over. As above, it has been cleaned up.